phalcon/Db/Dialect/Postgresql.zep

```diff
@@ -250,6 +259,14 @@ class Postgresql extends Dialect
         let sql .= join(",\n\t", createLines) . "\n)";
         if isset definition["options"] {
             let sql .= " " . this->getTableOptions(definition);
+            /**
+            * Check if there is a TABLE_COMMENT option
+            */
+            if fetch tableComment, definition["options"]["TABLE_COMMENT"] {
+                if tableComment {
+                    let indexSqlAfterCreate = " COMMENT ON TABLE " . table . " IS 'Our session logs';";
```

Comment:
Why here is hardcoded comment?

Reply:
set PGSQL table comment。
the sql is “comment on table table_name is 'xxx'; ”

Reply:
this is bug!